docs/developer/PROW_CI.md

```diff
@@ -64,12 +64,278 @@ The jobs run can be seen in PRs and in the links in the README.md file of OADP r
 
 OADP operator master branch is tested against the last 3 minor OCP releases. To Update an OCP version the project is tested against, see [Update OCP version](#update-ocp-version).
 
-### TODO Creating new release branch
+### Creating new release branches
 
-https://github.com/openshift/oadp-operator/issues/1227
+To create new OADP release branches (they must follow the pattern `oadp-major.minor`, [example](https://github.com/openshift/oadp-operator/tree/oadp-1.4)):
+- First, create CI files for each new repository branch.
+- Create the release branches in each one of related repositories of OADP, and OADP operator repository itself.
+- The new release branches must be updated:
+  - Downstream repositories release branches may not need updates (except OADP operator, which always need updates).
+  - Upstream repositories release branches need to be rebased.
+
+> **Note**: Try to always create release branches from default branches (usually called master).
+
+> **Note**: Documentation should live only in default branches. For example, for OADP operator, `docs/` and `blogs/` folders can be deleted in release branches.
 
 **TODO automate this process**
 
+#### Example: create CI files for OADP operator repository
+
+CI files for OADP operator (and all of its related repositories) live in https://github.com/openshift/release repository.
+
+For this example, lets say new release branch is `oadp-1.4`.
+
+The new release branch must be added to `core-services/prow/02_config/openshift/oadp-operator/_prowconfig.yaml` changing the following.
+```diff
+...
+   - includedBranches:
+     - master
+     - oadp-1.0
+     - oadp-1.1
+     - oadp-1.2
+     - oadp-1.3
++    - oadp-1.4
+     labels:
+...
+```
+
+Create `ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4.yaml`. To make it easier, copy the contents of `ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml` (if release branch was created from master branch), changing the following.
```

Comment:
What about that part of the file? I thought we want to make sure it's inline with the ocp version that the ci is running on?

```
build_root:
  image_stream_tag:
    name: builder
    namespace: konveyor
    tag: ubi9-v1.22.2
```

Reply:
I think this is responsibility of this issue https://github.com/openshift/oadp-operator/issues/1455

Reply:
On the above there is commend `As it is not valid anymore after...`, so either resurrect there or fix here.